doc/contributing/style_guides/otbn_style_guide.md

```diff
@@ -146,6 +146,168 @@ Prefer `.word` over alternatives such as `.quad`.
 
 The following guidelines address cryptography-specific concerns for OTBN assembly.
 
+### Handling secret shares
+
+The following guidelines were determined with the [help of the COCO-Alma tool](https://github.com/lowRISC/opentitan/tree/master/hw/ip/otbn/pre_sca/alma) and help protect against SCA attacks.
+
+**1. Do not overwrite a share with its counterpart share.**
+
+A register or memory location which contains one share must not be overwritten with its counterpart. The following lines would each violate this rule and cause transient leakage:
+```armasm
+/* Leakage (if w0, w1 contain two shares of a secret) */
+bn.and  w1, w0, w0
+bn.addi w1, w0, 0x0
+bn.mov  w1, w0
+bn.sel  w1, w0, w0, FG0.C
+```
+
+**2. Do not use multiple shares in a single instruction without masking.**
+
+Shares may interact with each other in the ALU and cause stable leakage at a gate or a register.
+The following lines would each violate this rule and cause stable leakage:
+```armasm
+/* Leakage (if w0, w1 contain two shares of a secret) */
+bn.add w4, w1, w0
+bn.xor w4, w1, w0
+```
+
+**3. Do not access shares within successive instructions.**
+
+By violating this rule, one can cause a transient leakage.
+Try to reorder the program or interleave the instructions.
+If this is not possible, use a dummy instruction.
+See following example as a dummy instruction usage.
+```armasm
+/* Leakage (if w0, w1 contain two shares of a secret) */
+bn.xor w4, w0, w0
+bn.xor w5, w1, w1
+
+/* No leakage */
+bn.xor w4, w0, w0
+bn.xor w9, w9, w9 /* dummy instruction */
+bn.xor w5, w1, w1
+```
+
+**4. Do not have shares even in different bits of a register.**
```

Comment:
Nit: I slightly struggled to understand this. Do we mean "Don't combine shares in a register, even in different bits" ?

Reply:
Yes, that's my understanding.